internal/config/config.go

```diff
@@ -81,6 +81,11 @@ func New(provider string) (h config.Handler, err error) {
 		return nil, err
 	}
 
+	// Setup Operations Config
+	if err := h.SetObject(adapter.OperationsKey, Operations); err != nil {
+		return nil, err
```

Comment:
Should this be using MeshKit style errors? https://docs.meshery.io/project/contributing/contributing-error

Reply:
I don't think so, the error is not encountered due to any network error. (As far as I understand)

Reply:
Network or not…. Doesn’t matter.  @Revolyssup  shine some light this, please.

Reply:
can someone pls help??